notebooks/Gemma+Gemini+Eval.ipynb

```diff
@@ -0,0 +1,8838 @@
+{
```

Comment:
<p>If you have received Gemini 1.5 API access, let's include that too? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/llmops-pipeline/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Not yet! maybe yes, but Gemini 1.5 Pro takes a long time for a single inference (at least 30 seconds I believe)

Reply:
Oh wow!